### DISCUSSION TITLE
Add bun to "Default Package Manager" option

### DISCUSSION THREAD

**Discussion by blaquewithaq** [nuxtrdev/nuxtr-vscode#60] [Ideas]
Good day, I was SUPER excited to come across this vscode extension in the Nuxt discord. I've been browsing the settings and noticed the "Default Package Manager" option was missing bun. Is there anyway you could add it when you get the chance.

Great job! I'm excited to use this for future projects.

Cheers!

**Comment by blaquewithaq** [comment]
Hmm. I just reread the release page and saw bun support was already added. My apologies. I just don't see the option on the settings list dropdown.

**Reply by adhamfarrag** [reply]
@Blaquewithaq Thanks for your kind words. Actually you're right about the option not being added to the schema. I'll post the fix shortly!

**Reply by blaquewithaq** [reply]
Awesome! Thank you so much for creating this, you all have done a great job and I'm excited to see where this progresses!

**Comment by adhamfarrag** [comment]
@Blaquewithaq Thanks :) Please, send all your feedback and thoughts ether here or from Project view in sidebar!
Fix release is released. Please, update and thanks for raising this.
